rescheduler.go

```diff
@@ -419,3 +426,16 @@ func updateSpotNodeMetrics(spotNodeInfos nodes.NodeInfoArray, pdbs []*policyv1.P
 func podID(pod *apiv1.Pod) string {
 	return fmt.Sprintf("%s/%s", pod.Namespace, pod.Name)
 }
+
+// Checks that the node lablels provided as arguments are in fact, sane.
+func validateArgs(OnDemandNodeLabel string, SpotNodeLabel string) error {
+	if len(strings.Split(OnDemandNodeLabel, "=")) > 2 {
+		return fmt.Errorf("the on demand node label: %s has more than one '='", OnDemandNodeLabel)
```

Comment:
Might be good to add some more explanation to this? Something like
```suggestion
		return fmt.Errorf("the on demand node label is not correctly formatted: expected '<label_name>' or '<label_name>=<label_value>', but got %s", OnDemandNodeLabel)
```